scripts/repackage.sh

```diff
@@ -124,7 +124,9 @@ NEWPKGJSONHASH=$(
       "src",
       "dist",
       "types",
-      "LICENSE*"
```

Comment:
perhaps a separate commit for this isn't that important, but some mention in the commit message seems worthwhile.